cmd/istioctl/main.go

```diff
@@ -280,14 +284,16 @@ and destination policies.
 				return err
 			}
 
+			objectNamespace := ""
+			ns, _ := handleNamespaces(objectNamespace)
```

Comment:
Can you use `""` directly here?

Reply:
will do, it's only for clarity.

Reply:
Understood. Maybe use `model.NamespaceAll` or `v1.NamespaceAll`?

Reply:
ok. I can do that for line 499 too.

Reply:
updated now.